### DISCUSSION TITLE
[08:41:29] Error while executing SQL query on database 'world': database disk image is malformed

### DISCUSSION THREAD

**Discussion by Raghu-M-S** [pawelsalawa/sqlitestudio#3863] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
The database file is either corrupted or encrypted and you need to open it with proper encryption plugin.
